core/iwasm/libraries/libc-emcc/libc_emcc_wrapper.c

```diff
@@ -495,6 +495,12 @@ emscripten_notify_memory_growth_wrapper(wasm_exec_env_t exec_env, int i)
     (void)i;
 }
 
+static void
+emscripten_sleep_wrapper(wasm_exec_env_t exec_env, int timeout_ms)
```

Comment:
Is the time unit ms? `man 3 sleep` in linux shows:
```C
NAME
       sleep - sleep for a specified number of seconds

SYNOPSIS
       #include <unistd.h>

       unsigned int sleep(unsigned int seconds);
```

Reply:
Yes this is confusing, but I believe the implementation is correct. Please see the emscripten documention here:

https://emscripten.org/docs/api_reference/emscripten.h.html#sleeping